tests/xtimer_now_irq/main.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2019 Freie Universität Berlin,
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       testing xtimer_mutex_lock_timeout function
+ *
+ *
+ * @author      Julian Holzwarth <julian.holzwarth@fu-berlin.de>
+ *
+ */
+
+#include <stdio.h>
+#include "xtimer.h"
+#include "irq.h"
+
+int main(void)
+{
+    puts("xtimer_now_irq test application.\n");
+
+    while (true) {
```

Comment:
Instead of an infinite loop, use a `count--` with count initialized with a sensible value.
Then print "SUCCESS" when getting out of the while loop and no error happened.